ksqldb-engine/src/main/java/io/confluent/ksql/planner/plan/QueryProjectNode.java

```diff
@@ -76,6 +76,7 @@
   private final boolean addAdditionalColumnsToIntermediateSchema;
   private final KsqlConfig ksqlConfig;
 
+  @SuppressFBWarnings("EI_EXPOSE_REP2")
```

Comment:
I suppressed this warning as I didn't see how adding this config added any mutability. Configs are effectively immutable, right?